examples/next/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
🤔 I guess this is the create-next-app default `README.md`.  I suppose down the line we can add a relevant readme to the various example apps identifying how envy was integrated, but for now we can probably ditch this?

Reply:
Yeah, I just ran the default `yarn create next-app` in the `examples` folder